web/src/lib/components/album-page/album-title.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import { updateAlbumInfo } from '@immich/sdk';
+  import { handleError } from '$lib/utils/handle-error';
+
+  export let id: string;
+  export let albumName: string;
+  export let isOwned: boolean;
+
+  $: newAlbumName = albumName;
+
+  const handleUpdateName = async () => {
+    if (newAlbumName === albumName) {
+      return;
+    }
+
+    try {
+      await updateAlbumInfo({
+        id,
+        updateAlbumDto: {
+          albumName: newAlbumName,
+        },
+      });
+      albumName = newAlbumName;
+    } catch (error) {
+      handleError(error, 'Unable to update album name');
+    }
+  };
```

Comment:
Here as well. Or did you decide on removing it completely?

Reply:
I think Alex deleted it in another pr

Reply:
Oh I see. Was that intentional or a bad merge?

Reply:
On purpose because it seemed a little noisy